apps/os/backend/utils/type-helpers.ts

```diff
@@ -1,12 +1,8 @@
-import type {
-  AnyRouter,
-  BuiltRouter,
-  MutationProcedure,
-  QueryProcedure,
-} from "@trpc/server/unstable-core-do-not-import";
-import type { AnyTRPCRootTypes, AnyTRPCRouter } from "@trpc/server";
+import type { MutationProcedure, QueryProcedure } from "@trpc/server/unstable-core-do-not-import";
```

Comment:
```suggestion
```

Reply:
oh damn i guess we have lint warnings off. will turn them on in another pr